src/content/app/tools/blast/components/blast-app-bar/BlastAppBar.tsx

```diff
@@ -53,8 +54,13 @@ const BlastAppBar = () => {
     return <Link to={urlFor.speciesSelector()}>Change</Link>;
   }, []);
 
+  const appBarProps = {
+    appName: AppName.TOOLS,
```

Comment:
Why is a generic name like Tools given to the BLAST app?

Reply:
I do not know. Andrea's XDs for BLAST say so 🤷‍♂️